Posts/2021/02/ScriptingGuy.0002.GetDateOfYesterday.md

```diff
@@ -0,0 +1,185 @@
+# Getting Yesterday's Date
+
+**Q:** How can I get yesterday's date?
+
+**A:** You can use a combination of the `Get-Date` cmdlet and .NET Time/Date methods.
+
+First, let's look at dates in PowerShell and .NET
+Then we can look at how to calculate yesterday and use that in your scripts.
+
+## Dates in PowerShell
+
+Let's start by looking at how you can deal with dates and times.
+As you probably know, PowerShell contains the `Get-Date` cmdlet.
+This cmdlet returns a .NET **System.DateTime** object.
+
+Using the `Get-Date` cmdlet, you can get any date and time, and either display it or store it in a variable. 
+To get today's date. you could do this:
+
+```powershell
+# Get the current date
+Get-Date
+# Store the date in a variable then view it
```

Comment:
Can you add a new line between examples for better readability?

Reply:
Will add